websites/B/bilibili/presence.ts

```diff
@@ -103,37 +103,50 @@ presence.on("UpdateData", async () => {
         delete presenceData.endTimestamp;
       }
 
-      title = document.querySelector("#viewbox_report > h1");
-      user = document.querySelector(
-        "#v_upinfo > div.u-info > div > a.username"
-      );
+      if (huodong_title != null) {
+        title = document.querySelector("#viewbox_report > h1 > span");
+      } else if (hudong_title != null) {
+        title = document.querySelector(
+          "#viewbox_report > h1 > span.tit.tr-fix"
+        );
+      } else {
+        title = document.querySelector("#viewbox_report > h1");
+      }
+      if (multi_user != null) {
+        user = document.querySelector(
+          "#member-container > div:nth-child(1) > div.panel > div > a"
+        );
+      } else {
+        user = document.querySelector(
+          "#v_upinfo > div.u-info > div > a.username"
+        );
+      }
       presenceData.details = title.innerText;
-      presenceData.state = "By user: " + user.innerText;
+      presenceData.state = user.innerText;
+
       //播放历史
     } else if (document.location.pathname == "/account/history") {
       presenceData.startTimestamp = browsingStamp;
-      presenceData.details = "Viewing history";
+      presenceData.details = "Their history";
```

Comment:
`Viewing their history` might be better
Viewing history or Their history is both not really good.

Reply:
> `Viewing their history` might be better
> Viewing history or Their history is both not really good.

i saw that youtube uses "their history" so i copied it